### ISSUE TITLE
Several Create Compatibility Recipes Broken - 1.18.2

### ISSUE THREAD

**Issue by DocWhoSonicDrive** [CannoliCatfish/project-rankine#357]
Create updating to version 0.5.1 has changed the name of certain crushed ores. This has caused some _compatibility recipes to not load_.
I am playing with several mods, but create is version 0.5.1.b and Rankine is 1.3.8
I am on Minecraft version 1.18.2

Here, I have attached a text file of the 6 recipes I found to not load (there may be more) and the missing items that caused them not to load.
[ListOfMissingCreateRankineRecipes.txt](https://github.com/CannoliCatfish/project-rankine/files/11607216/ListOfMissingCreateRankineRecipes.txt)

**In short,**

- 'create:crushed_gold_ore' became 'create:crushed_raw_gold'
- 'create:crushed_iron_ore' became 'create:crushed_raw_iron'
- 'create:crushed_copper_ore' became 'create:crushed_raw_copper'
- 'create:crushed_zinc_ore' became 'create:crushed_raw_zinc'

and it broke recipes.
